### PR TITLE
Finalize javascript_nodejs/01.console-echo

### PR DESCRIPTION
**Pull request by benbrown**
- [x] Is numbered and named correctly and following the spec
- [x] All files have been spell checked
- [x] All style cop/ linting issues have been addressed
- [x] Include all DCR changes
- [x] Does not use mainDialog unless that file actually extends Dialog/ ComponentDialog
- [x] Compiles with latest packages
- [x] Runs per instructions in README
- [x] README to include
  - [ ] msbot clone steps
  - [ ] Tools install/ uninstall
  - [x] Concise set of steps to run the sample

---

Not applicable:
- [ ] Explicit state save (no state in this sample)
- [ ] onTurnError at adapter (console adapter doesn't have this)
- [ ] Method name changes begin, end etc (no dialogs)
- [ ] Add commented out code to use blob storage (no storage of any kind used)
- [ ] Update to look for production/ local (bot configuration)
- [ ] Include .env or appsettings.json or web.config with botFilePath and botFileSecret (no config is present)
- [ ] C# sample -- Include .bot file in the build output
- [ ] All samples include the latest botfile (no "secretKey" field; replaced with "padLock"). Use latest MSBOT off the tools MyGet feed to generate the bot file
- [ ] Deployment scripts folder has been cleaned up and only includes msbot export files

### CODE REVIEW

**Comment by benbrown** [comment]
I think this is ready - it is the simplest one and so a lot of the checklist items don't apply.

File: samples/javascript_nodejs/01.console-echo/index.js [hunk 1]
```diff
@@ -15,13 +12,17 @@ require('dotenv').config({ path: ENV_FILE });
 // We are using the ConsoleAdapter, which enables a bot you can chat with from within your terminal window.
 const adapter = new ConsoleAdapter();
 
+// Import our bot class.
+const EchoBot = require('./bot');
```
Comment (sgellock) [inline samples/javascript_nodejs/01.console-echo/index.js]: change to:
const { EchoBot } = require('./bot');

File: samples/javascript_nodejs/01.console-echo/README.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ This sample shows how to create a simple echo bot that you can talk to from the
     ```bash
     git clone https://github.com/microsoft/botbuilder-samples.git
     ```
-- In a terminal, navigate to javascript_nodejs/01.console-echobot
+- In a terminal, navigate to samples/javascript_nodejs/01.console-echo
     ```bash
-    cd javascript_nodejs/01.console-echobot
+    cd samples/javascript_nodejs/01.console-echo
     ```
     - Point to the MyGet feed
```
[the remark this answers is not in the document]
Comment (benbrown) [reply samples/javascript_nodejs/01.console-echo/README.md]: Will clean these out as I go!

File: samples/javascript_nodejs/01.console-echo/index.js (hunk 1 above)
Comment (benbrown) [reply samples/javascript_nodejs/01.console-echo/index.js]: Love named imports.

**Comment by benbrown** [comment]
I'm not sure what to do with the msbot clone steps or deployment scripts in this case. is a .bot file appropriate here?

**Pull request merged**